src/test/java/ru/yandex/clickhouse/integration/RowBinaryStreamTest.java

```diff
@@ -150,6 +206,77 @@ public void writeTo(ClickHouseRowBinaryStream stream) throws IOException {
         Assert.assertEquals(rs.getDouble("float32"), 123.456);
         Assert.assertEquals(rs.getDouble("float64"), 42.21);
 
+        final Date[] dateArray = (Date[]) rs.getArray("dateArray").getArray();
+        Assert.assertEquals(dateArray.length, dates1.length);
+        for (int i = 0; i < dateArray.length; i++) {
+            // expected is Date at start of the day in server timezone
+            DateTime dt = new DateTime(dates1[i].getTime())
```

Comment:
@genuss , Regarding the tests failure I mentioned in https://github.com/yandex/clickhouse-jdbc/pull/108,  the code actually worked correctly, the problem I found was in this test.
Expected Date value from Date field must be at start of the day in server timezone. Please check if I've done it correctly.
Also I've reproduced this at CI by setting timezone to ClickhHouse server different from UTC (that was enough) and  from client application(tests).